### PR TITLE
The final commit

### PR DESCRIPTION
**Pull request by lucashicks1**
# Final Commit...

The final commit to get to 420 commits. Changing the most important part of the game.... the readme

**#NOTE: DON'T WORRY THIS PR IS BREAKING MAIN WITH A README CHANGE**

### CODE REVIEW

**Review by JTrenerry — APPROVED** [review]
It's sad im approving a failing PR but oh well

**Comment by lucashicks1** [comment]
> It's sad im approving a failing PR but oh well

I'll wait till the studio for the final commit, hopefully main isn't broken by then

**Comment by JTrenerry** [comment]
> > It's sad im approving a failing PR but oh well
> 
> I'll wait till the studio for the final commit, hopefully main isn't broken by then

If it is broken by studio ill merge my fix, but team 5 should fix it

File: README.md [hunk 1]
```diff
@@ -19,6 +19,17 @@ To get started with the game, please follow the outline provided [here](https://
...
 - [Game Wiki](https://github.com/UQcsse3200/2023-studio-1/wiki)
 
+## Course Description
+
+This game was developed in just 8 weeks, with work being broken up into 4 different 2-week long sprints. The goals for each sprint were as follows:
+
+- Sprint 1 - Minimum Viable Product
+- Sprint 2 - Core Game Functionality
+- Sprint 3 - Full Game Delivery
+- Sprint 4 - Polished Game
```
Comment (vilnor) [inline README.md:29]: "polished game" sounds about right

**Pull request merged**